docs/pages/blog/material-ui-v6-is-out.md

```diff
@@ -0,0 +1,395 @@
+---
+title: Material UI v6 is out now 🎉
+description: Material UI v6 ships with support for CSS variables, container queries, and advanced color schemes, as well as improved runtime performance and a reduced bundle size.
+date: 2024-08-09T00:00:00.000Z
```

Comment:
```suggestion
date: 2024-08-22T00:00:00.000Z
```

Reply:
Lets set it to Monday and target to release that day

```suggestion
date: 2024-08-26T00:00:00.000Z
```